fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -153,27 +152,6 @@ object FAPI {
     yield Collection.draftContent(collection, setOfContent, snapContent)}
 
   /**
-   * Fetches content for the given backfill query. The query can be manipulated for different
-   * requirements by providing adjustment functions. The results then have their facia metadata
-   * resolved using the collection information.
-   */
-  @deprecated
-  def backfill(backfillQuery: String, collection: Collection,
```

Comment:
> Do you need to remove this in this PR?
Yes, we deprecated `backfill` in 0.68, frontend and apps upgraded fapi, no one is using that method anymore, so it should be removed because there's another one that hides the implementation details of backfill.